src/main/java/Gene/task/TaskList.java

```diff
@@ -0,0 +1,90 @@
+package Gene.task;
+
+import Gene.Gene;
+import Gene.GeneException;
+
+import java.util.ArrayList;
+
+import static Gene.command.MarkCommand.isNumeric;
+
+public class TaskList {
+    private ArrayList<Task> toDoList = new ArrayList<>();
+    private final Storage fileData = new Storage();
+    public TaskList() {
+        toDoList = fileData.loadTasksFromFile();
+    }
+
+    private void saveTasksToFile() {
+        fileData.saveTasksToFile(toDoList);
+    }
+
+    public void addTask(Task newTask) {
+        toDoList.add(newTask);
+        saveTasksToFile();
+        Gene.printLineSeparation();
+        System.out.println("Got it. I've added this task:");
+        System.out.println("  " + newTask);
+        System.out.println("Now you have " + toDoList.size() + " tasks in the list.");
+        Gene.printLineSeparation();
```

Comment:
Perhaps you can seperate the lines into different sections with blank spaces to improve readability. Same goes for other similar occurrences in this class.